### PR TITLE
Fix User Guide

### PR DESCRIPTION
**Pull request by ChuaZenKhoon**
Add hyperlinks to each command(#99 also done)
Issues:
1)#113
2)#147
3)#144, #119, #116, #110, #103
4)#146
5)#145
6)#108
7)#141
8)#133
9)#132,#100

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -296,27 +456,48 @@ GUI upon successful addition command <br><br>
...
     <img src="images/ui/toggle/hide.png" width="452.5" height="369.5"><br><br>
   </box>
 
-</panel>
+### Finding Contacts by Name: <code>?</code>
+[back to top](#table-of-contents)
+
+Finds all contacts in the Hi:Re app that has names matching the words entered.
+
+Format: <code> > ? (name) </code><br>
```
Comment (ljy0422) [inline docs/UserGuide.md]: The format is wrong

File: docs/UserGuide.md [hunk 2]
```diff
@@ -82,9 +106,11 @@ Breakdown of commands:
 
    <img src="images/ui/startUp.png" width="452.5" height="369.5"><br>
```
Comment (ljy0422) [inline docs/UserGuide.md:107]: The start up image doesn't seem to be updated? #95

File: docs/UserGuide.md [hunk 3]
```diff
@@ -12,6 +12,17 @@
...
+  2) [Login into Account](#login-into-account-codelogincode)
+  2) [Logout from Account](#logout-from-account-codelogoutcode)
+  2) [Adding a Contact](#adding-a-contact-codecode)
+  2) [Deleting a Contact](#deleting-a-contact-code-code)
+  3) [Editing a Contact](#editing-a-contact-codecode)
+  4) [Toggling the display](#toggling-the-display-codecode)
+  5) [Finding Contacts by Name](#finding-contacts-by-name-codecode)
+  5) [Listing all matching information](#listing-all-matching-information-codelscode)
+  6) [Undoing a Command](#undoing-a-command-codeundocode)
+  7) [Redoing a Command](#redoing-a-command-coderedocode)
```
Comment (ljy0422) [inline docs/UserGuide.md]: These hyperlinks don't work.

**Review by ljy0422 — COMMENTED** [review]
Some small issues need to be fixed.

**Comment by ChuaZenKhoon** [comment]
#99 woops not #95 but also done

File: docs/UserGuide.md (hunk 1 above)
Comment (ChuaZenKhoon) [reply docs/UserGuide.md]: woops thx!

File: docs/UserGuide.md (hunk 3 above)
Comment (ChuaZenKhoon) [reply docs/UserGuide.md]: i dont think u can test the hyperlink by the code file, need to open in intellij or on the website then can check

**Pull request merged**